pro/src/screens/OfferIndividual/StocksEvent/adapters/__specs__/serializers.spec.ts

```diff
@@ -41,10 +41,10 @@ describe('screens::StockEvent::serializers:serializeStockEventList', () => {
     expect(serializedData).toStrictEqual([expectedApiStockEvent])
   })
 
-  it('should serialize data for stock event without "bookingLimitDatetime"', async () => {
+  it('should serialize data for stock event with "bookingLimitDatetime" even if not provided', async () => {
```

Comment:
On pourrait indiquer qu'on met la valeur de beginningDate time ? `should fill bookingLimitDatetime with beginningDatetime when not provided`